07.bowling_object/game.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  STRIKE = 'X'
+
+  def initialize(marks)
+    @marks = marks.split(',')
+  end
+
+  def score
+    frames = create_frames
+    frames.each_with_index.map do |frame, idx|
+      if idx < 9
+        frame.score + bonus(frames, frame, idx)
+      else
+        frame.score
+      end
+    end.sum
+  end
+
+  private
+
+  def create_frames
+    frames = Array.new(9) do
+      marks = @marks.first == STRIKE ? [@marks.shift] : @marks.shift(2)
```

Comment:
`STRIKE`の定義はShotにもっていって、

```suggestion
      marks = Shot.strike?(@marks.first) ? [@marks.shift] : @marks.shift(2)
```

みたいにするのが一手ですかね〜。

Reply:
こちらをもとに修正してみましたありがとうございます！
ご確認よろしく願いします。